Jint/project.json

```diff
@@ -1,18 +1,17 @@
 {
-  "version": "1.0.0-*",
+  "version": "2.7.2-*",
   "dependencies": {
-    "System.Diagnostics.Contracts": "4.0.1-beta-23409",
-    "System.Dynamic.Runtime": "4.0.11-beta-23409",
-    "System.Linq": "4.0.1-beta-23409",
-    "System.Linq.Expressions": "4.0.11-beta-23409",
-    "System.Reflection.TypeExtensions": "4.0.1-beta-23409",
-    "System.Text.RegularExpressions": "4.0.11-beta-23409"
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Dynamic.Runtime": "4.0.0",
+    "System.Reflection.TypeExtensions": "4.0.0",
+    "System.Text.RegularExpressions": "4.0.0",
+    "System.Collections": "4.0.0",
```

Comment:
How did you figure out all these libraries were missing? Were they included by another facade package?